R/check-cardinalities.R

```diff
@@ -1,33 +1,33 @@
 #' Test if the relation between two tables of a data model meet the requirements
 #'
-#' @description All `check_cardinality()` functions test, if the `pk_column` is a unique key for `parent_table` and if
+#' @description All `check_cardinality()` functions test if `pk_column` is a unique key for `parent_table` and if
```

Comment:
I prefer splitting long sentences. Can you please use bullets or an enumeration? For example:

```suggestion
#' @description All `check_cardinality()` functions test the following conditions:
#' 1. Is `pk_column` a unique key for `parent_table`?
```

Reply:
Certainly, I am happy to do that (did not dare to restructure the original text too much :) )